File: mvppresenter/liba/src/main/java/me/hutcwp/liba/TestFragment.java
package me.hutcwp.liba;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import hut.cwp.core.MvpFragment;

public class TestFragment extends MvpFragment<TestPresenter, ITest> {

    public TestFragment() {
    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        return inflater.inflate(R.layout.frgament_test, container, false);
    }

}

File: live/src/main/java/com/hutcwp/live/TestFragment.kt
package com.hutcwp.live

import android.app.Fragment
import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup


/**
 *  author : kevin
 *  date : 2021/12/12 3:06 PM
 *  description :
 */
class TestFragment : Fragment() {


    override fun onCreateView(inflater: LayoutInflater?, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        return inflater?.inflate(R.layout.live_layout_test,container,false)
    }
}